config/_default/menus/menus.en.toml

```diff
@@ -4,8 +4,8 @@
   weight = 10
 
 [[main]]
-  name = "Officers"
-  url = "/officers/"
+  name = "People"
+  url = "/people/"
```

Comment:
This looks like a good change assuming ACL is not imposing some standard on us where we have to call it Officers

Reply:
@d-ataman and @mammadhajili suggestion, maybe revisit at meeting?